src/modules/xml/producer_xml.c

```diff
@@ -1411,6 +1601,10 @@ static void on_start_element( void *ctx, const xmlChar *name, const xmlChar **at
 		on_start_filter( context, name, atts );
 	else if ( xmlStrcmp( name, _x("transition") ) == 0 )
 		on_start_transition( context, name, atts );
+	else if ( xmlStrcmp( name, _x("chain") ) == 0 )
+		on_start_chain( context, name, atts );
+	else if ( xmlStrcmp( name, _x("link") ) == 0 )
+		on_start_link( context, name, atts );
```

Comment:
We need to update `src/modules/xml/mlt-xml.dtd` with the new `chain` and `link` elements.

Reply:
done